# FORKS
Finding Orderings Robustly using K-means and Steiner trees

Recent advances in single cell RNA-seq technologies have provided researchers with unprecedented details of transcriptomic variation across individual cells. However, it
has not been straightforward to infer differentiation trajectories from such data. Here, we present Finding Orderings Robustly
using K-means and Steiner trees (FORKS), an algorithm that pseudo-temporally orders cells and thereby infers bifurcating state
trajectories. FORKS, which is a generic method, can be applied to both single-cell or bulk differentiation data. It is a semi-supervised approach, in that it requires
the user to specify the starting point of the time course. We systematically benchmarked FORKS and 6 other pseudo-time estimation algorithms on 5 benchmark datasets, and
found it to be more accurate, more reproducible, faster and more memory-efficient than existing methods for pseudo-temporal ordering. Another major
advantage of our approach is that the algorithm requires no hyperparameter tuning.
